emails/tests/views_tests.py

```diff
@@ -284,10 +289,33 @@ def test_notification_type_complaint(self):
         with MetricsMock() as mm:
             response = _sns_notification(json_body)
         assert response.status_code == 200
-        mm.assert_incr_once("fx.private.relay.email_complaint")
-        assert len(self.caplog.records) == 1
-        record = self.caplog.records[0]
-        assert record.msg == "complaint_received"
+
+        self.user.profile.refresh_from_db()
+        assert self.user.profile.auto_block_spam is True
```

Comment:
praise: nice! I thought we we already doing this? Or we were only doing it for *bounces* that contained the term "spam" in the reason?